frontend/src/scenes/feature-flags/FeatureFlagProjects.tsx

```diff
@@ -0,0 +1,115 @@
+import { LemonTable, LemonTableColumns } from 'lib/lemon-ui/LemonTable'
+import { LemonButton, LemonModal, LemonModalProps, LemonSelect } from '@posthog/lemon-ui'
+import { IconArrowRight } from 'lib/lemon-ui/icons'
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
+import { featureFlagLogic } from './featureFlagLogic'
+import { organizationLogic } from '../organizationLogic'
+
+const records: Record<string, string>[] = [{ project: 'project-1-staging', flag_status: 'enabled' }]
+
+export function CopyFeatureFlagModal({
+    overlayRef,
+    contentRef,
+}: Pick<LemonModalProps, 'overlayRef' | 'contentRef'>): JSX.Element {
+    const { featureFlag, isProjectsModalOpen } = useValues(featureFlagLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { closeProjectsModal } = useActions(featureFlagLogic)
+
+    return (
+        <LemonModal
+            overlayRef={overlayRef}
+            contentRef={contentRef}
+            isOpen={isProjectsModalOpen}
+            onClose={closeProjectsModal}
```

Comment:
When I try to "X" it out, it doesn't work :0 but "cancel" and clicking outside of the modal does

Reply:
I'm moving the form out of the modal so no longer an issue :)